typedbSchemaBuilder/MyType.py

```diff
@@ -1,19 +1,19 @@
-class my_type:
+class MyType:
```

Comment:
`My` is not a good part for a class name if this project is supposed to be not your own test project. Could it be just `Type`?